### PR TITLE
Issue #77: Add "Exit OpenLoco" to game menu

### PR DESCRIPTION
**Pull request by seifer7**
Issue #77 : Add "Exit OpenLoco" to game menu

- Adds "Exit OpenLoco" to the in-game Load/Save menu.
- Changes existing "Quit Game" menu item to "Quit to menu"
- Changes existing "Quit Scenario Editor" menu item to "Quit to menu"

This is all matched to what OpenRCT2 says.

### CODE REVIEW

File: data/language/de-DE.yml [hunk 1]
```diff
@@ -116,7 +116,9 @@ strings:
   102: Objekt im Wege
   103: Spiel laden
   104: Spiel speichern
+  # TODO: requires updating. English now reads: 'Quit to menu'
```
Comment (petergaal) [inline data/language/de-DE.yml:119]: Did you miss similar change in sk-SK.yml file?

Comment (seifer7) [reply data/language/de-DE.yml:119]: Good spotting. Apparently I missed it.
Once I'm back at my PC I'll change it.
Thank You!

**Comment by seifer7** [comment]
So sorry. I tried to fix that missing language file using a Git app on my phone and it screwed up the permissons on two files that I didn't even tell it to commit =/

File: data/language/de-DE.yml (hunk 1 above)
Comment (seifer7) [reply data/language/de-DE.yml:119]: Fixed.

**Comment by seifer7** [comment]
Okay. This is done now and ready for review/commit.

**Comment by AaronVanGeffen** [comment]
Thanks for the PR. It looks good, but as you've already changed the editor strings as well, I'd like to merge this PR after #427. Once that's in, we can rebase this PR and add the changes to the relevant menu.

File: data/language/en-GB.yml [hunk 2]
```diff
@@ -116,8 +116,8 @@ strings:
   102: Object in the way
   103: Load Game
   104: Save Game
-  105: Quit Scenario Editor
-  106: Quit Game
```
Comment (AaronVanGeffen) [inline data/language/en-GB.yml:120]: Note to self: these strings appear to be used by more than just the top toolbars. They should not be changed.

I'll change this when I rebase this.

**Pull request merged**